pkg/client/retrieve.go

```diff
@@ -65,6 +64,13 @@ func (c *SonobuoyClient) RetrieveResults(cfg *RetrieveConfig) (io.Reader, <-chan
 		return nil, nil, errors.Wrap(err, "failed to get the name of the aggregator pod to fetch results from")
 	}
 
+	// DEBUG; dont assume Linux os
```

Comment:
Need to still have some logic to choose the right tar command.

I think I just need to GET the aggregator pod, see which node it is on, GET the node and check its labels for the OS.

Better idea?

Reply:
The only other thing I can think of right now is to have a utility script or something with the same name that exists inside both the Linux and Windows images that would take care of forming the correct tar command -  similarly to how we're now relying on the Dockerfile to know how to run the aggregator. I don't think that would work though due to the differences in client/server versions. It seems much more likely for the `retrieve` command that you could be using a newer client version against an older server version.

I think checking the node labels is probably the way to go.